crates/subspace-networking/src/behavior/persistent_parameters.rs

```diff
@@ -329,13 +329,13 @@ impl NetworkingParametersManager {
                     }
 
                     // Verify checksum
-                    let actual_hash = blake3_hash(encoded_bytes);
-                    let expected_hash = &remaining_bytes[..mem::size_of::<Blake3Hash>()];
-                    if actual_hash != expected_hash {
+                    let actual_checksum = blake3_hash(encoded_bytes);
+                    let expected_checksum = &remaining_bytes[..mem::size_of::<Blake3Hash>()];
```

Comment:
Why not use `BLAKE3_HASH_SIZE` in all places?

Reply:
Very good question. Actually I'd like to make both that constant and similar for Blake2 private. We don't have standalone constants for sizes of other data structures, so it seems more consistent to not use them here either. Also this way size and type are inherently linked and usage of type in IDE reveals all relevant places.

Maybe I'm just trying to rationalize irrational things :man_shrugging:

Reply:
> Also this way size and type are inherently linked and usage of type in IDE reveals all relevant places.

This was my initial theory: dependency chain management.